OrbitQt/orbitmainwindow.cpp

```diff
@@ -201,50 +201,28 @@ static QWidget* CreateSpacer(QWidget* parent) {
   return spacer;
 }
 
-static QAction* CreateDummyAction(const QIcon& icon, QObject* parent) {
+[[nodiscard]] static QAction* CreateDummyAction(const QIcon& icon, QObject* parent) {
   auto* action = new QAction(icon, "", parent);
   action->setDisabled(true);
   return action;
 }
 
-static QIcon GetIcon(const std::string& icon_name) {
+[[nodiscard]] static QIcon GetIcon(const std::string& icon_name) {
   return QIcon(Path::JoinPath({Path::GetIconsPath(), icon_name}).c_str());
```

Comment:
*wish-mode on* :innocent: : Since you're revamping the icons, it would be great to change this function such that it reads the icons from the embedded resource. As far as I've seen you're embedding all icons anyway.

But this is extending the scope quite a bit, so feel free to ignore that.

Bit if you decide to do that then the icons-deployment-logic can/has to be removed. There is a copy target in `OrbitQt/CMakeLists.txt` and a `self.copy(...)`-command in conanfile.py's `package`-routine. Searching for `icons` in the mentioned files should point you to the right spots.

Reply:
> _wish-mode on_ 😇 : Since you're revamping the icons, it would be great to change this function such that it reads the icons from the embedded resource. As far as I've seen you're embedding all icons anyway.
> 
> But this is extending the scope quite a bit, so feel free to ignore that.
> 
> Bit if you decide to do that then the icons-deployment-logic can/has to be removed. There is a copy target in `OrbitQt/CMakeLists.txt` and a `self.copy(...)`-command in conanfile.py's `package`-routine. Searching for `icons` in the mentioned files should point you to the right spots.

I wanted to do that, then I saw that it's a bite more of an effort and decided to leave it for now, let's have a look at that in the future.

Reply:
sounds good! :+1: